crates/jwk/src/multicodec.rs

```diff
@@ -47,6 +47,9 @@ impl JWK {
             ssi_multicodec::BLS12_381_G2_PUB => {
                 crate::bls12381g2_parse(k).map_err(FromMulticodecError::Bls12381G2Pub)
             }
+            ssi_multicodec::JWK_JCS_PUB => {
```

Comment:
Can you add a test checking that its able to decode a JWK encoded with the JWK_JCS_PUB multicodec?

Reply:
Done!